lib/package.gi

```diff
@@ -1429,7 +1455,8 @@ BindGlobal( "LoadPackage_ReadImplementationParts",
         elif IsBound( info.BannerString ) then
```

Comment:
So in this code, the first `if` could be changed to 
```gap
if UserPreference( "ShortBanners" ) then
  bannerstring := ShortPackageBannerString(info);
elif IsBound( info.BannerFunction ) then
  ...
```

Thoughts?

Reply:
So custom banners would be complete ignored when `ShortBanners` is set to `true`? I just looked at some packages and, for example, `curlInterface` adds the versions of various libraries to the banner. I guess that this information is printed for debugging and in particular to have this information available in bug reports? If so, I think ignoring the custom banner would be suboptimal.

Reply:
A clean interface would maybe be that packages can specify additional information which gets included in the banner instead of letting `BannerFunction` patch `DefaultPackageBannerString`. And of course this additional information could then come in two variants, a long and a short version. But this would be beyond what I had planned for this PR :D

Reply:
The curl version can also be obtained in other ways.

And I'd try not to overthink this problem. Honestly, the whole quest for "short banners" is already extremely fringe. Most people will either not mind the long banner, or just turn banners off. Having a middle ground is not something I can see a lot of people having much interest in (I might be totally wrong, of course)